pipelines/build/prTester/README.md

```diff
@@ -0,0 +1,127 @@
+# PR Testers
```

Comment:
This README is very helpful, but probably hard to find. We might need a summary in the top-level README with a link.

Reply:
I was planning to add exactly that into the FAQ as a minimum

Reply:
@sxa Could  you add a link to this readme on your faq?

Reply:
Of course, but at the moment neither this PR or mine have gone in so we should probably do it separately, and based on a comment I'm about to make I don't want to pre-empt the URL for when I merge my initial FAQ :-)

Reply:
Its been added